pkg/querier/queryrange/roundtrip.go

```diff
@@ -334,7 +334,7 @@ func NewLabelsTripperware(
 	retryMiddlewareMetrics *queryrange.RetryMiddlewareMetrics,
 	splitByMetrics *SplitByMetrics,
 ) (queryrange.Tripperware, error) {
-	queryRangeMiddleware := []queryrange.Middleware{StatsCollectorMiddleware(), NewLimitsMiddleware(limits)}
```

Comment:
double checking: here the `StatsCollectorMiddleware()` is removed. Doesn't that cause us to lose the statistics? :thinking:

Reply:
The labels query does not return any stats, and the unintended added middleware would only log a warning:
https://github.com/grafana/loki/blob/main/pkg/querier/queryrange/stats.go#L100

Reply:
got it, ty!